.github/workflows/regression_tests_docker.yml

```diff
@@ -0,0 +1,56 @@
+name: Run Regression Tests on Docker
+
+on:
+  push
+    # run every day at 5:15am
```

Comment:
nit: remove comments

Reply:
Yes, this is temporary. Will update this once we have the PR approved. We don't have a good way to run this